modules/decrypt-node/src/verify_stream.ts

```diff
@@ -284,7 +284,9 @@ export class VerifyStream extends PortableTransformWithType {
       needs(isVerified, 'Invalid Signature')
       callback()
     } catch (e) {
-      callback(e)
+      if (e instanceof Error) {
+        callback(e)
+      }
```

Comment:
This is problematic. JS can throw whatever it likes, but if it did throw, there was still an error. An err-back function does not care what the error is.  This may be a TS is being ... too good at its job?

TL;DR; you can't just swallow an error. Even if that error is super unlikely :(